docs/source/examples/performance_modeling_output.txt

```diff
@@ -17,44 +17,24 @@ W_{burn} : [ 0.487     0.486     0.485     0.485    ]  [lbf] segment fuel burn
 W_{fuel} : [ 1.94      1.46      0.97      0.485    ]  [lbf] fuel weight
 
          | Mission/FlightSegment/AircraftP/WingAero
-     C_D : [ 0.00783   0.00782   0.00781   0.0078   ]        drag coefficient
-     C_L : [ 0.47      0.469     0.468     0.467    ]        lift coefficient
        D : [ 4.87      4.86      4.85      4.85     ]  [lbf] drag force
-      Re : [ 7.56e+05  7.56e+05  7.56e+05  7.56e+05 ]        Reynold's number
 
-Constants
----------
+Sensitivities
+-------------
      | Aircraft/Fuselage
-   W : 100                                         [lbf]       weight
+   W : +0.25            weight
 
      | Aircraft/Wing
-   A : 27                                                      aspect ratio
-   S : 190                                         [ft**2]     surface area
-\rho : 1                                           [lbf/ft**2] areal density
+   S : +0.68            surface area
+\rho : +0.48            areal density
+   A : -0.31            aspect ratio
 
+Next Largest Sensitivities
```

Comment:
this seems an awkward title when the table above is "Sensitivities". Should the table above be "Largest Sensitivities"?